lib-ce/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -744,3 +756,11 @@ listener_identifier(Protocol, ListenOn) ->
         ID ->
             ID
     end.
+
+%% TODO Maybe redesign this api.
+restart_emqx_management(Identifier) ->
+    emqx_ctl:print("Listener ~p not support restart!~n", [Identifier]).
```

Comment:
```suggestion
    emqx_ctl:print("Management listener restart is not supported~n", []).
```

Reply:
Is this planned for the next release?
I think it is possible to implement a real command to restart the HTTP listener. Why are we including it in this PR?